pkg/endpoint/bpf.go

```diff
@@ -601,21 +601,29 @@ func (e *Endpoint) regenerateBPF(owner Owner, epdir, reason string) (uint64, err
 	defer func() {
 		if err != nil {
 			e.Mutex.Lock()
+			epLogger := e.getLogger()
+			epLogger.WithError(err).Error("destroying BPF maps due to" +
+				"errors during regeneration")
 			if createdPolicyMap {
+				epLogger.Info("removing endpoint PolicyMap")
 				os.RemoveAll(e.PolicyMapPathLocked())
 				e.PolicyMap = nil
 			}
 
 			if createdIPv6IngressMap {
+				epLogger.WithField(logfields.BPFMapName, e.IPv6IngressMapPathLocked()).Info("removing endpoint CIDR map")
```

Comment:
All of this info log messages are very low level. Why do we need these in a non debug environment?

Reply:
Fixed.